modules/learn/pages/architecture-diagrams.adoc

```diff
@@ -1,51 +1,139 @@
-= Architecture Diagrams
+= List of Architecture Diagrams
 
 [abstract]
 A quick reference to architecture diagrams in Couchbase documentation.
 
 Throughout the Couchbase documentation-set, explanations of the architecture of Couchbase Server are supported diagrammatically.
 Each diagram is co-located with its textual description. This page provides a quick-reference, whereby each diagram can be directly accessed.
 
-The following diagrams are provided:
+Diagrams are provided for the following topics. Click on the thumbnails or other links to access the full-size diagrams and accompanying, detailed descriptions.
 
-* Couchbase Server supports _multidimensional scaling_, whereby services can be distributed and resourced with the greatest flexibility, across the designated nodes of a cluster.
-This is explained and diagrammatically represented in xref:learn:services-and-indexes/services/services.adoc#setting-up-services[Setting Up Services].
+== Multidimensional Scaling
 
-* The Couchbase _Data Service_ is the most fundamental of all Couchbase services, providing access to data in memory and on disk.
-The diagram in xref:learn:services-and-indexes/services/data-service.adoc[Data Service] shows its architecture, and the interactions of its inner components.
+Couchbase Server supports _multidimensional scaling_, whereby services can be distributed and resourced with the greatest flexibility, across the designated nodes of a cluster.
 
-* The Couchbase _Query Service_ supports the querying of data by means of the N1QL query language.
-Its architecture and the query processing-sequence it supports are explained by the diagrams in xref:learn:services-and-indexes/services/query-service.adoc[Query Service].
+[#cb_cluster_with_services_production_thumbnail]
+xref:learn:services-and-indexes/services/services.adoc#setting-up-services[image:services-and-indexes/services/cbClusterWithServicesProduction.png[,220,align=left]]
+
+This is explained in xref:learn:services-and-indexes/services/services.adoc#setting-up-services[Setting Up Services].
+
+== Data Service
+
+The Couchbase _Data Service_ is the most fundamental of all Couchbase services, providing access to data in memory and on disk.
+
+[#data_service_architecture_thumbnail]
+xref:learn:services-and-indexes/services/data-service.adoc[image:services-and-indexes/services/dataServiceArchitecture.png[,220,align=left]]
+
+See xref:learn:services-and-indexes/services/data-service.adoc[Data Service] for a description of the Data Service' architecture, and the interactions of its inner components.
```

Comment:
```suggestion
See xref:learn:services-and-indexes/services/data-service.adoc[Data Service] for a description of the Data Service architecture, and the interactions of its inner components.
```

Reply:
@tonyjhillman this one didn’t get applied. I think the period at the end was a typo?